website/content/en/docs/reference/metrics.md

```diff
@@ -34,6 +38,30 @@ Number of nodeclaims disrupted in total by Karpenter. Labeled by reason the node
 Number of nodeclaims created in total by Karpenter. Labeled by reason the nodeclaim was created and the owning nodepool.
 - Stability Level: STABLE
 
+### `operator_nodeclaim_status_condition_transitions_total`
+The count of transitions of a nodeclaim, type and status. Labeled by the type, reason, and status.
+- Stability Level: BETA
```

Comment:
Any opinions if these metrics should be alpha or beta? We kept them as Beta since their original metric was Beta

Reply:
Agreed offline, these will stay beta.